### ISSUE TITLE
Add the preset for sidewalks

### ISSUE THREAD

**Issue by GoogleCodeExporter** [MarcusWolschon/osmeditor4android#131]
```
Would it be possible for the tag for sidewalks to be added, so that it comes up 
in the list of keys when I'm typing the key in?

http://wiki.openstreetmap.org/wiki/Key:sidewalk

Possible values to fill in are: both, left, right and no. (Best to leave out 
yes, as it's ambiguous as you don't know which sides of the roads have a 
sidewalk).
```

Original issue reported on code.google.com by `shaunmcd...@gmail.com` on 21 Sep 2012 at 8:41
- Blocked on: #134
- Merged into: #134
